app/src/main/java/com/example/bookwormadventuresdeluxe2/NotificationUtility/APIService.java

```diff
@@ -0,0 +1,20 @@
+package com.example.bookwormadventuresdeluxe2.NotificationUtility;
+
+
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.Headers;
+import retrofit2.http.POST;
+
+public interface APIService
+{
+    @Headers(
+            {
+                    "Authorization:key=AAAAxQez4bg:APA91bHXkCnrOfTnVcmkfxC-9hqMrvvNJdHb_qEcG4-j0YxbUaHFq9kB6pzHkU4XJSJhyTt7lpXgCyXyO8jJRokJIKErI9JZVGkCRV1nIBUb1f1intxwHai1RCTrhObXX6nQ8rK4AHDN",
```

Comment:
Should this key be kept in secrets.xml?

Reply:
The attributes here take constants only, wasn't able to use context.getString() from secrets.xml. 
Added CloudMessagingServerKeyConstant.java to .gitignore. Pls see image:
![cloudServerKey](https://user-images.githubusercontent.com/44795394/100156541-56a30c00-2e66-11eb-8207-ef989ec8477c.PNG)